services/gateway/apis/http-version3/methods/transactionsStatistics.js

```diff
@@ -26,6 +27,27 @@ module.exports = {
 		limit: { optional: true, type: 'number', min: 1, max: 100, default: 10, pattern: /^\b((?:[1-9][0-9]?)|100)\b$/ },
 		offset: { optional: true, type: 'number', min: 0, default: 0, pattern: /^\b([0-9][0-9]*)\b$/ },
 	},
+	get schema() {
+		const transactionSchema = {};
+		transactionSchema[this.swaggerApiPath] = { get: {} };
+		transactionSchema[this.swaggerApiPath].get.tags = this.tags;
+		transactionSchema[this.swaggerApiPath].get.summary = 'Requests transaction statistics';
+		transactionSchema[this.swaggerApiPath].get.description = getSwaggerDescription({
+			rpcMethod: this.rpcMethod,
+			description: 'Returns transaction statistics',
+		});
+		transactionSchema[this.swaggerApiPath].get.parameters = transformParams('transactions', this.params);
```

Comment:
Please fix the `description` and allowed values for `interval` param. It currently displays irrelevant information.